File: RemoveDublicates/RemDub.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        # Создаю переменную для элементов в массиве
        i = 1
        # Создаю цикл итераций
        while i < len(nums):
            # Сравнение элементов. Если элементы в списке одинаковы, элементы удаляются
            if nums[i-1] == nums[i]:nums.pop(i)
            # Либо инкрементирую элементы
            else:
                i += 1
            # Возвращаю длину мутированного массива как в условии
        return len(nums)

File: RemoveDublicates/RemDub.js
var removeDuplicates = function(nums) {
    // Cоздаю переменную для сравнения
    let i = 0
    // Создаю цикл и новую вторую переменную для сравнения элементов в массиве
    for (let j = 1; j < nums.length; j++) {
        // Если элементы не равны, то передвигаю первый вперед по итерации
        if (nums[j] !== nums[i]) {
            i+=1
            // Ставлю их вместе
            nums[i] = nums[j]
        } 
    } 
    return i + 1
};